.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: 
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: sh
+    steps:
+    - uses: actions/checkout@v2
+    - name: Lint and Test
+      run: |
```

Comment:
What is the difference between putting this all sequentially in a single run statement vs separate run statements?

Reply:
Nothing much.Its just short-hand notation.